Source/Core/Core/HW/GCMemcard/GCMemcard.h

```diff
@@ -189,8 +189,8 @@ struct HeaderData
   // 4 bytes at 0x0018: SRAM language
   Common::BigEndianValue<u32> m_sram_language;
 
-  // 4 bytes at 0x001c: ? almost always 0
-  std::array<u8, 4> m_unknown_2;
+  // 4 bytes at 0x001c: VI DTV status register value (u16 from 0xCC00206E)
```

Comment:
Maybe add the documentation as a comment:

```
this register allows software to read the status of two i/o pins
```

Not the most helpful but might provide a hint to someone in the future?  I always wonder what might happen if those documentation websites die.  I feel like the source should be self documented.

Reply:
I don't think that's necesary for several reasons:
 - YAGCD is well known and available both as online and offline documentation, plus the description provided is `VISEL - don't care`.
 - This register is already documented in [libogc/video](https://github.com/devkitPro/libogc/blob/master/libogc/video.c).
 - It's also documented in Dolphin code itself [UVIDTVStatus](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/HW/VideoInterface.h)